modules/swagger-models/src/main/java/io/swagger/models/Operation.java

```diff
@@ -209,16 +213,53 @@ public void addParameter(Parameter parameter) {
         }
         this.parameters.add(parameter);
     }
-
+    @Deprecated
     public Map<String, Response> getResponses() {
         return responses;
```

Comment:
this needs to return responsesObject (converted to Map) or responses depending on null status of the two. see how this is achieved for responseSchema and schema in response